src/MatrixClient.cc

```diff
@@ -315,20 +314,20 @@ MatrixClient::sendRoomMessage(mtx::events::MessageType ty,
 
                 if (!json.isObject()) {
                         qDebug() << "Send message response is not a JSON object";
-                        emit messageSendFailed(roomid, txnId);
                         return;
                 }
 
                 auto object = json.object();
 
                 if (!object.contains("event_id")) {
                         qDebug() << "SendTextMessage: missing event_id from response";
-                        emit messageSendFailed(roomid, txnId);
                         return;
                 }
 
                 emit messageSent(object.value("event_id").toString(), roomid, txnId);
         });
+
+        incrementTransactionId();
```

Comment:
This can be removed.

Reply:
Yeah, git was very unhelpful when it showed the merge conflict for this file